.github/workflows/security-scans.yml

```diff
@@ -27,3 +42,35 @@ jobs:
 
     - name: Gitleaks - detecting hardcoded secrets
       uses: zricethezav/gitleaks-action@v1.1.4
+
+  bandit:
+    name: Detect python security issues
+    runs-on: ubuntu-latest
+    needs: [changes]
+
+    steps:
+    - name: Checkout git repository 🕝
+      if: needs.changes.outputs.backend == 'true'
```

Comment:
no not really, because the workflow are required to pass by the branch protection mechanism. We cannot skip the workflows altogether, so we have to live with this today 😄